LEAF_Request_Portal/form.php

```diff
@@ -3170,21 +3170,27 @@ public function query($inQuery)
     public function getDisabledIndicatorList($disabled)
     {
         $vars = array(':disabled' => (int)$disabled);
-        $res = $this->db->prepared_query('SELECT * FROM indicators
-											LEFT JOIN categories USING (categoryID)
-						                    WHERE indicators.disabled >= :disabled
-						    					AND categories.disabled = 0
-						    				ORDER BY name', $vars);
+        $strSQL = "SELECT indicatorID, name, format, description, categories.categoryName, ".
+                    "indicators.disabled FROM indicators ".
+                    "LEFT JOIN categories USING (categoryID) ".
+                    "WHERE indicators.disabled >= :disabled ".
+                    "AND categories.disabled = 0 ".
+                    "ORDER BY name";
+
+        $res = $this->db->prepared_query($strSQL, $vars);
 
         $data = array();
         foreach ($res as $item)
         {
             $temp = array();
+            $delDate = $item['disabled'] + 30*24*60*60; //30 days from timestamp
+            $delDate = date("m/d/Y",$delDate);
             $temp['indicatorID'] = $item['indicatorID'];
```

Comment:
Bad juju.  Never reuse a variable within the same function for different things.  Especially when you are changing the type.  Once a variable is declared within a scope, while its value can change, what it is never can.